### ISSUE TITLE
Implement the markdownUnfurlImagePlugin into the file transformer pipeline

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/devhub-app-web#218]
as mentioned in #217 the code has been authored and tested but not implemented. Opening a new issue to implement the file transformer plugin that was created in #217 AFTER #213 has been merged!

_Originally posted by @patricksimonian in https://github.com/bcgov/devhub-app-web/issue_comments#issuecomment-449160137_

**Comment by patricksimonian** [comment]
@orinevares @lukegonis another good issue to work on. Hold of on it for now until #213 has been merged.

**Comment by patricksimonian** [comment]
this is already to be worked on !
